pages/common/gmssl.md

```diff
@@ -0,0 +1,36 @@
+# gmssl
+
+> GmSSL is a crypto toolkit supporting SM1, SM2, SM3, SM4, SM9, and ZUC/ZUC256
+> More information: <http://gmssl.org/>.
+
+- Generate an SM3 hash for a file:
+
+`gmssl sm3 <file>`
```

Comment:
```suggestion
`gmssl sm3 {{path/to/file}}`
```

Please change this throughout the page.

Reply:
Fixed; resubmitting pull request. Thanks!